integrated_channels/cornerstone/models.py

```diff
@@ -318,3 +319,37 @@ class CornerstoneCourseKey(models.Model):
 
     class Meta:
         app_label = 'cornerstone'
+
+
+class CornerstoneAPIRequestLogs(IntegratedChannelAPIRequestLogs):
+    """
+     A model to track basic information about every API call we make from the integrated channels.
+    """
+    user_agent = models.CharField(max_length=255)
+    user_ip = models.GenericIPAddressField(blank=False, null=False)
```

Comment:
is it confirmed that this field will always have data?

Reply:
Good catch! I think no, CSOD right now is _pull_ not _push_ so an HTTP call typically includes standard information like user agent and IP. But I think it should be optional since there are many endpoints in CSOD client which are POST and won't have IP